corehq/apps/userreports/templates/userreports/partials/report_content.html

```diff
@@ -0,0 +1,53 @@
+{% load i18n %}
+
+<div class="report-content">
+  {% if report_table.data_source %}
+    <div class="table-responsive">
+      <table
+        class="table table-striped datatable"
+        id="report_table_{{ report.slug }}"
+      >
+        <thead>
+          <tr>
+            {% for column in report_table.data_source.columns %}
+              <th>{{ column.header }}</th>
+            {% endfor %}
+          </tr>
+        </thead>
+        <tbody>
+          {% if report_table.data_source.data %}
+            {% for row in report_table.data_source.data %}
+              <tr>
+                {% for key, value in row.items %}
+                  <td>{{ value|safe }}</td>
```

Comment:
nit: Can you share the intent for use of safe here?

Reply:
Same question - is `value` being escaped somewhere? I'd expect that this value is *not* safe since reports typically display user-generated content.

Reply:
Same question for all the usages of `safe` in this file.